src/services/pocket-relayer.ts

```diff
@@ -238,6 +241,7 @@ export class PocketRelayer {
                 origin: this.origin,
                 data,
                 pocketSession: this.pocketSession,
+                sticky: preferredNodeAddress === (await getAddressFromPublicKey(relayResponse.proof.servicerPubKey)),
```

Comment:
`sticky` in the metricsRecorder is defined as a boolean but here you're sending it the address?

I think the better measure would be: `SUCCESS` or `FAILURE` or `NONE`

Success: if stickiness was desired and successful

Failure: if stickiness was desired but unsuccessful

None: not desired